lightning-invoice/src/utils.rs

```diff
@@ -629,8 +629,11 @@ fn filter_channels<L: Deref>(
 
 			include_channel
 		})
-		.map(route_hint_from_channel)
-		.collect::<Vec<RouteHint>>()
+                .collect::<Vec<ChannelDetails>>();
```

Comment:
just a minor issue: you're indenting with spaces, but this files use tabs

Reply:
Fixed in latest push - thanks!